include/boost/url/impl/url_base.ipp

```diff
@@ -1337,6 +1337,166 @@ end() const noexcept ->
     return iterator(v_, true);
 }
 
+auto
+url_base::
+segments_type::
+erase( url_base::segments_type::iterator pos ) noexcept ->
```

Comment:
I believe you can simply write `iterator pos` instead of `url_base::segments_type::iterator pos`

Reply:
Indeed.